release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -2256,7 +2256,7 @@ In the following tables, features are marked with the following statuses:
 |General Availability
 
 |Pod-level bonding for secondary networks
-|Technology Preview
+|link:https://docs.openshift.com/container-platform/4.10/release_notes/ocp-4-10-release-notes.html#ocp-4-10-30-pod-level-bonding-ga[GA]
```

Comment:
You can't direct link to docs.openshift. Please add another section to the 4.12 release notes.